blockchain/stake/internal/tickettreap/immutable_test.go

```diff
@@ -98,7 +98,7 @@ func TestImmutableSequential(t *testing.T) {
 
 	// Insert a bunch of sequential keys while checking several of the treap
 	// functions work as expected.
-	expectedSize := uint64(0)
+	var expectedSize int
```

Comment:
Same.  I'll stop commenting, but essentially every one of these needs to be reverted to ensure the same semantics across archs.